app/Migrations/Version20220607125416.php

```diff
@@ -0,0 +1,180 @@
+<?php
```

Comment:
Do we need all these tables? or the command table will be enough?

Reply:
command table should be enough, I'll fix